packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -117,6 +122,29 @@ function useSimpleActions() {
     })
   }
 
+  if (superchargeRewards.length > 0) {
+    actions.push({
+      text: t('superchargeNotificationBody'),
+      icon: boostRewards,
+      priority: SUPERCHARGE_AVAILABLE_PRIORITY,
+      callToActions: [
+        {
+          text: t('superchargeNotificationStart'),
+          onPress: () => {
+            ValoraAnalytics.track(HomeEvents.notification_select, {
+              notificationType: NotificationBannerTypes.supercharge_available,
+              selectedAction: NotificationBannerCTATypes.accept,
+            })
+            navigate(Screens.ConsumerIncentivesHomeScreen)
+            ValoraAnalytics.track(RewardsEvents.rewards_screen_opened, {
+              origin: RewardsScreenOrigin.RewardAvailableNotification,
```

Comment:
nit: it's easy to forget to track this for each call site that navigates to the `ConsumerIncentivesHomeScreen` 
We could require the origin as a screen param to enforce this and trigger that event in the screen itself.

But more generally it would be nice to avoid having both the generic "screen" opened event and custom screen opened like this one.

Reply:
Yeah, we just wanted to know where the opens were coming from with the default screen open doesn't have, although I'm not sure that's important anymore 🤔  100% agree on passing a param though, it's indeed very easy to forget to add it.